src/Notes/WC_Admin_Notes_Insight.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * WooCommerce Admin: Do you need some inspiration?
+ *
+ * Adds a note to ask the client if they need some inspiration.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Insight.
+ */
+class WC_Admin_Notes_Insight {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-insight';
```

Comment:
@pmcpinto Will more insights be added in the future?  If so, let's rename this to be more specific, otherwise you can ignore this comment @becdetat.

Reply:
Done in 0f9379669af5ed0706605dbe95f9a24969739264